contrib/kind-dual-stack-conversion.sh

```diff
@@ -0,0 +1,290 @@
+#!/usr/bin/env bash
+
+convert_kubelet() {
+ echo "Converting kubelet on nodes $NODES"
+ for n in $NODES; do
+   docker exec $n cp /var/lib/kubelet/config.yaml /var/lib/kubelet/config.yaml.bak
+   # kubelet already has a featureGate configured so we need to ammend a new one
+   # NOTE: if there weren't feature gates configured we have to change this
+   docker exec $n sed -i '/featureGates/a\  IPv6DualStack: true' /var/lib/kubelet/config.yaml
+   # Restart kubelet
+   docker exec $n systemctl restart kubelet
+  echo "Converted kubelet on node $n"
+ done
+}
+
+convert_cni() {
+  # from kind hack/ci/e2e-k8s.sh
+  # Modify current OVN config
+  ORIGINAL_OVNCONFIG=$(kubectl get -oyaml -n=ovn-kubernetes configmap/ovn-config)
+  echo "Original OVN config:"
+  echo "${ORIGINAL_OVNCONFIG}"
+  # Patch it
+  FIXED_OVNCONFIG=$(
+    printf '%s' "${ORIGINAL_OVNCONFIG}" | sed \
+      -e "s#^  net_cidr.*#&\,${SECONDARY_CLUSTER_SUBNET}#" \
+      -e "s#^  svc_cidr:.*#&\,${SECONDARY_SERVICE_SUBNET}#" \
+  )
+  echo "Patched OVN config:"
+  echo "${FIXED_OVNCONFIG}"
+  printf '%s' "${FIXED_OVNCONFIG}" | kubectl apply -f -
+  # restart ovnkube-master
+  # FIXME: kubectl rollout restart deployment leaves the old pod hanging 
+  # as workaround we delete the master directly
+  kubectl -n ovn-kubernetes delete pod -l name=ovnkube-master
+  # restart ovnkube-node
+  kubectl -n ovn-kubernetes rollout restart daemonset ovnkube-node
+  kubectl -n ovn-kubernetes rollout status daemonset ovnkube-node
+  echo "Updated CNI"
+}
+
+convert_k8s_control_plane(){
+  # Kubeadm installs apiserver and controller-manager as static pods
+  # one the configuration has changed kubelet restart them
+  API_CONFIG_FILE="/etc/kubernetes/manifests/kube-apiserver.yaml"
+  CM_CONFIG_FILE="/etc/kubernetes/manifests/kube-controller-manager.yaml"
+
+  # update all the control plane nodes
+  for n in $CONTROL_PLANE_NODES; do
+    echo "Converting control-plane on node $n"
+    # kube-apiserver backup config file
+    docker exec $n cp ${API_CONFIG_FILE} /kind/${API_CONFIG_FILE}.bak
```

Comment:
I think these cp commands are failing because of double slashes:

API_CONFIG_FILE="/etc/kubernetes/manifests/kube-apiserver.yaml"

/kind/${API_CONFIG_FILE}.bak